### PR TITLE
Add PSKoans to Learning Resources documentation

### PR DESCRIPTION
**Pull request by vexx32**
# PR Summary

Add a link to PSKoans for the learning resources.

## PR Context

PSKoans is a PowerShell module being developed to create an interactive learning experience for new PowerShell users. It assumes some basis of CS knowledge but no specific PowerShell experience.

**Related question:** 

This "Recommended Training and Reading" heading doesn't seem quite the right way to put it? To me, at least, that implies that these resources have been personally vetted by Microsoft staff in order to be able to give a recommendation on the basis of some concrete assessment -- which at least at first glance doesn't seem to be likely for all of the items listed?

I think we may want to update the wording for this heading as well. Maybe something more neutral such as "Educational Resources" might be slightly more appropriate? Open to suggestions, I'm not completely sure on that one. 🙂 

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

**Comment by vexx32** [comment]
@PoshChan please retry all

**Comment by PoshChan** [comment]
@vexx32, you are not authorized to request a rebuild

**Comment by SteveL-MSFT** [comment]
@vexx32 https://dot.net is currently down which breaks the boot strapper, it's being worked on

File: docs/learning-powershell/README.md [hunk 1]
```diff
@@ -102,6 +102,7 @@ Note that all bash commands should continue working on PowerShell session.
 - [Learn PowerShell Video Library][idera-learn-powershell] from Idera
 - [PowerShell 5 How-To Videos][script-guy-how-to] by Ed Wilson
 - [PowerShell Documentation](https://docs.microsoft.com/powershell)
+- [PSKoans](https://aka.ms/pskoans)
```
Comment (SteveL-MSFT) [inline docs/learning-powershell/README.md]:
```suggestion
- [Interactive learning with PSKoans](https://aka.ms/pskoans)
```

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]
I think it's more meaningful to indicate it's learning interactively

File: docs/learning-powershell/README.md (hunk 1 above)
Comment (vexx32) [reply docs/learning-powershell/README.md]: Yeah, I wasn't sure how to word it. That does work well, thanks! Brain is only 60% on today it seems. 😄

**Comment by SteveL-MSFT** [comment]
@PoshChan please retry all

**Review by SteveL-MSFT — APPROVED** [review]

**Comment by PoshChan** [comment]
@SteveL-MSFT, successfully started retry of `PowerShell-CI-static-analysis, PowerShell-CI-Windows, PowerShell-CI-macOS, PowerShell-CI-Linux`

**Pull request merged**